src/dlite-json.c

```diff
@@ -1116,11 +1116,12 @@ DLiteJsonFormat dlite_jstore_loads(JStore *js, const char *src, int len)
       t += jsmn_count(v) + 2;
     }
   }
+  retval = format;
 
  fail:
   if (tokens) free(tokens);
   if (uri) free(uri);
-  return format;
+  return retval;
```

Comment:
A bit confusing why retval = format? Seems like a confusing naming convention. Why not just return the format??

Reply:
See above